backend/infrahub/computed_attribute/tasks.py

```diff
@@ -186,7 +190,11 @@ async def process_jinja2(
         for id_filter in computed_macro.node_filters:
             filters = {id_filter: object_id}
             nodes = await service.client.filters(
-                kind=computed_macro.kind, prefetch_relationships=True, populate_store=True, **filters
+                kind=computed_macro.kind,
+                branch=branch_name,
+                prefetch_relationships=True,
```

Comment:
it's not directly related to this PR but while looking at the code recently I was wondering if we need `prefetch_relationships=True` here

Reply:
Yes we need the relationships in this context if the Jinja2 template references a related node. In the future it would perhaps be cleaner if we could extract the variables used by the Jinja2 template and compile a more minimalistic query using that instead of sending such verbose queries. Then we'd also be able to get rid of the schema refresh here.